edit.php

```diff
@@ -48,6 +48,12 @@
         fb("Failure on $key => ". print_r($value, TRUE) ." for $edit_user");
       }
       fb("Success on $key => ". print_r($value, TRUE) ." for $edit_user");
+    } elseif (is_array($new_user_data[strtolower($key)])) {
```

Comment:
(I'm basically wondering if this would remove certain characters like emoji that a user DID add/change and then not update something because it _thinks_ there's no change when really there was.)